.travis.yml

```diff
@@ -172,7 +172,10 @@ before_deploy:
     - docker login --email=$DOCKER_HUB_EMAIL --username=$DOCKER_HUB_USERNAME --password=$DOCKER_HUB_PASSWORD
 
     # Set latest tag only for HEAD of master branch
-    - if [ "$TRAVIS_BRANCH" = "master" ]; then docker push $DOCKER_REPOSITORY:latest; fi
+    - if [ "$TRAVIS_BRANCH" = "master" ]; then
+        docker tag $DOCKER_REPOSITORY:pr $DOCKER_REPOSITORY:latest;
```

Comment:
why we tag this with `pr` in the first place ?
btw are you sure you have to retag to be able to push as specified tag?